index.html

```diff
@@ -169,5 +169,50 @@ <h4 class="profile-tittle">Skills</h4>
             </article>
         </div>
     </section>
+
+    <section class="form-section">
+        <div class="header-img"></div>
+        <div class="form-container container">
+            <h1 class="tittle">Get started</h1>
+            <p class="para para2">
+                Lorem ipsum dolor sit amet, consectetur adipiscing
+                elit, sed do eiusmod tempor incididunt ut labore
+                et dolore magna.
+            </p>
```

Comment:
- To improve your site's accessibility, I'd advise you not to have more than one `h1` tag in your HTML code. Please kindly fix this 👍